numba/core/types/misc.py

```diff
@@ -141,17 +141,25 @@ def key(self):
 class CPointer(Type):
     """
     Type class for pointers to other types.
+
+    Attributes:
+        dtype : The type current pointer represents.
```

Comment:
```suggestion
        dtype : The pointee type.
```
maybe a better description.